lightstep/instrument.py

```diff
@@ -177,12 +178,27 @@ def _flush_worker(self, connection):
 
     def _construct_report_request(self):
         """Construct a report request."""
+        report = None
         with self._mutex:
             report = ttypes.ReportRequest(self._runtime, self._span_records,
                                           self._log_records)
             self._span_records = []
             self._log_records = []
-            return report
+
+        for log in report.log_records:
+            if log.stable_name is not None:
+                log.stable_name = str(log.stable_name)
+            if log.payload_json is not None:
```

Comment:
The hit from synchronously JSON-encoding payloads is pretty severe, especially if the caller logs large objects.